vsokolovsky/01.HTML/Homework_2.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Python 01</TITLE>
+    </HEAD>
+    <BODY>
+        <h1>HOW-TO: Программа на Python, Часть 1</h1>
+        <ul>
+            <li>Программа на Python — часть 1</li>
+            <li>Программа на Python — часть 2</li>
+            <li>Программа на Python — часть 3</li>
+        </ul>
+        <h2>Содержание</h2>
+        <h3>HOW-TO: Программа на Python, Часть 1</h3>
+        <ul>
+            <li>Наша первая программа</li>
+            <li>Пример цикла For</li>
+        </ul>
+        <p>Из множества существующих языков программирования Python — один из самых лёгких в изучении.
+        Он создан в конце 80-х годов XX века, но с тех пор он сильно изменился. Хотя Python входит в
+        состав большинства дистрибутивов Linux, на него часто не обращают внимания при выборе языка
+        для изучения. В этой статье мы рассмотрим создание консольных приложений, а в следующей —
+        программирование графических интерфейсов (GUI). Давайте создадим простую программу прямо
+        сейчас.</p>
+        <h2>Наша первая программа</h2>
+        <p>Наберём код в текстовом редакторе (например, в gedit). Потом мы разберёмся, что делает каждая
+        строка и пойдём дальше.</p>
+        <p>Введите эти 4 строки:</p>
+<pre> 
```

Comment:
Could you please remove this empty pre, and add one extra line in pre section below

Reply:
@pluhin Ok, i'll do it.